### PR TITLE
Add Metrics section to Scaling Airbyte doc

### PR DESCRIPTION
**Pull request by sophia-wiley**
Added Recommended Metrics section to [Scaling Airbyte](https://docs.airbyte.com/operator-guides/scaling-airbyte/).

Related to this [issue](https://github.com/airbytehq/airbyte/issues/12910).

### CODE REVIEW

File: docs/operator-guides/scaling-airbyte.md [hunk 1]
```diff
@@ -66,3 +66,42 @@ The advice here is best-effort and by no means comprehensive. Please reach out o
 
 If you've been running Airbyte in production and have more tips up your sleeve, we welcome contributions!
 
+## Recommended Metrics
+Airbyte supports exporting built-in metrics to Datadog or [OpenTelemetry](https://docs.airbyte.com/operator-guides/collecting-metrics/)
+
+### Key Metrics
```
Comment (Amruta-Ranade) [inline docs/operator-guides/scaling-airbyte.md:72]: Convert this section into a table

File: docs/operator-guides/scaling-airbyte.md [hunk 2]
```diff
@@ -66,3 +66,42 @@ The advice here is best-effort and by no means comprehensive. Please reach out o
...
+* `oldest_pending_job_age_secs`
+    * This metric shows how long a pending job waits before it is scheduled. If a job is in pending state for a long time, more workers may be required.
```
Comment (Amruta-Ranade) [inline docs/operator-guides/scaling-airbyte.md:74]: After reformatting this section as a table, you can delete the phrase "this metric" and start the descriptions with the verb ("shows...")

File: docs/operator-guides/scaling-airbyte.md [hunk 3]
```diff
@@ -66,3 +66,42 @@ The advice here is best-effort and by no means comprehensive. Please reach out o
...
+`job_created_by_release_stage\[“alpha”\] = 1;`  
+`job_created_by_release_stage\[“beta”\] = 1;`  
+`job_failed_by_release_stage\[“alpha”\] = 1;`  
+`job_succeeded_by_release_stage\[“beta”\] = 1;`  
+
+**Note:** Each job has a source and destination, so each metric is counted twice — once for source and once for destination.
+
+:::
+
+### Additional Recommended Metrics
```
Comment (Amruta-Ranade) [inline docs/operator-guides/scaling-airbyte.md:101]: same comments as above (re: table)

File: docs/operator-guides/scaling-airbyte.md (hunk 1 above)
Comment (sophia-wiley) [reply docs/operator-guides/scaling-airbyte.md]: Done :)

File: docs/operator-guides/scaling-airbyte.md (hunk 2 above)
Comment (sophia-wiley) [reply docs/operator-guides/scaling-airbyte.md]: Done :)

**Review by Amruta-Ranade — APPROVED** [review]

**Comment by sophia-wiley** [comment]
@xiaohansong and @davinchia -- The Recommended Metrics section has been merged to the master branch.

**Pull request merged**